surch/logger.py

```diff
@@ -58,9 +58,8 @@ def configure():
         sys.exit('Failed retrieving log file path ({0}).'.format(str(ex)))
     log_dir = os.path.dirname(os.path.expanduser(log_file))
     if os.path.isfile(log_dir):
-        sys.exit('File {0} exists - log directory cannot be created '
-                 'there. please remove the file and try again.'.format(
-            log_dir))
+        sys.exit('File {0} exists - log directory cannot be created there.'
```

Comment:
@nirfuchs, maybe we can implement a reasonable assertion logic here?

https://github.com/cloudify-cosmo/repex/compare/rearchitecture makes it pretty easy I think.